source/tutorials/atacama-soils.rst

```diff
@@ -173,7 +180,7 @@ and ``--p-trunc-len-r``, but that is not a requirement.
 
 At this stage, you will have artifacts containing the feature table and
 corresponding feature sequences. You can generate summaries of those as
-follows.
```

Comment:
How do you feel about this:
`At this stage, you will have artifacts containing the feature table,`
`corresponding feature sequences, and DADA2 denoising stats. You`
`can generate summaries of these as follows.`

Reply:
Sorry, couldn't figure out how to set this up as a multi-line suggestion - GH is complaining because there's a deleted line included. :shrug:

Reply:
To do a multiline comment, click and drag in the gutter next to the line numbers (same place you click for doing a single-line comment. Once the comment is started, suggesting a change will cover all lines selected). I applied your change below.